### PR TITLE
Dev to Main 20221222

### PR DESCRIPTION
**Pull request by hvgazula**

### CODE REVIEW

**Review by zkokaja — APPROVED** [review]
It all looks good to me. The one thing I would change is to remove `requirements.yml`, because now we have multiple environment files and it's confusing which takes precedence and which is most up-to-date. I recommend sticking with the `247-main` repo which I recently updated and tested.

File: requirements.yml [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+    - wcmatch==8.4.1
+    - wcwidth==0.2.5
+    - webencodings==0.5.1
+    - websocket-client==1.3.3
+    - whisper==1.0
+    - widgetsnbextension==3.6.1
+    - xmltodict==0.13.0
+    - xxhash==3.0.0
+    - yarl==1.8.1
+prefix: /home/hgazula/.conda/envs/247-main
```
Comment (zkokaja) [inline requirements.yml:209]: Needs to be removed.

**Review by VeritasJoker — APPROVED** [review]

**Comment by VeritasJoker** [comment]
@hvgazula do you want to change the yaml file then merge or submit it as an issue after merging?

**Comment by hvgazula** [comment]
Go ahead and merge this PR. I will do it in next week's PR.

**Pull request merged**